spec/helpers/profiles_helper_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+RSpec.describe ProfilesHelper do
+  describe "#jobseeker_status" do
+    let(:profile) { double("profile") }
+    let(:personal_details) { double("personal_details") }
```

Comment:
Suggestion:
I believe you can save some stub setup in the "before" block by inlining it here.

```suggestion
    let(:personal_details) { double("personal_details", right_to_work_in_uk: right_to_work_in_uk) }
```

I would also suggest defaulting to `instance_double` over `double` when possible, as it ensures the stubbed methods exist in the real object interface, helping to avoid the specs becoming outdated with time.
But you may have some specific reason that I may be missing here.

Reply:
Ah nice, thanks @scruti. Done!